config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
```

Comment:
Dự án sử dụng mysql. Không dùng sqllite. Tham khảo https://stackoverflow.com/questions/3585/create-a-new-ruby-on-rails-application-using-mysql-instead-of-sqlite